test/jbuilder/schema/template_test.rb

```diff
@@ -138,13 +138,15 @@ def setup
 
   test "collections" do
     assert_equal({description: "test", type: :array, items: {id: {description: "test", type: :integer}, title: {description: "test", type: :string}}}, json.articles(articles, :id, :title))
-    assert_equal({description: "test", type: :array, items: {id: {description: "test", type: :integer},
-                                                             title: {description: "test", type: :string},
-                                                             body: {description: "test", type: :string},
-                                                             created_at: {description: "test", type: :string, format: "date-time"},
-                                                             updated_at: {description: "test", type: :string, format: "date-time"},
-                                                             user_id: {description: "test", type: :integer}}},
-      json.articles(articles))
+    assert_equal({
+      description: "test", type: :array,
+      items: {id: {description: "test", type: :integer},
+      title: {description: "test", type: :string},
+      body: {description: "test", type: :string},
+      created_at: {description: "test", type: :string, format: "date-time"},
+      updated_at: {description: "test", type: :string, format: "date-time"},
+      user_id: {description: "test", type: :integer}}
+    }, json.articles(articles))
```

Comment:
This test depends on an article having a specific set of instance variables and previously got around `@user` by defining a method on the article singleton… which is pretty advanced for some fixture code and irked me. I swapped to `- ["user"]` in `attribute_names`, but that's not the best either.